geode-core/src/integrationTest/java/org/apache/geode/distributed/internal/membership/gms/fd/GMSHealthMonitorJUnitTest.java

```diff
@@ -337,19 +336,11 @@ public void testSuspectMembersCalledThroughSuspectThread() throws Exception {
    */
   @Test
   public void testSuspectMembersNotCalledThroughSuspectThreadBeforeTimeout() {
-
     installAView();
 
     gmsHealthMonitor.suspect(mockMembers.get(1), "Not responding");
 
-    try {
-      // suspect thread timeout is 200 ms
-      Thread.sleep(100l);
```

Comment:
Does removing this not make the test flaky? Seems like it would have the potential to fail on that verify. I have the same question about removing lines 370 and 443 (in the old file's numbering)

Reply:
No, the suspect state we're checking is set synchronously in the suspect call above.  I'm not sure why this sleep was ever here.  InstallView also synchronously sets all the state needed for the test, making the old sleeps at 370 and 443 similarly pointless.